src/main/java/com/launchdarkly/client/PollingProcessor.java

```diff
@@ -0,0 +1,62 @@
+package com.launchdarkly.client;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class PollingProcessor implements UpdateProcessor {
+  private static final Logger logger = LoggerFactory.getLogger(PollingProcessor.class);
+
+  private final FeatureRequestor requestor;
+  private final LDConfig config;
+  private final FeatureStore store;
+  private volatile boolean initialized = false;
+  private ScheduledExecutorService scheduler = null;
+
+  PollingProcessor(LDConfig config, FeatureRequestor requestor) {
+    this.requestor = requestor;
+    this.config = config;
+    this.store = config.featureStore;
+  }
+
+  @Override
+  public boolean initialized() {
+    return initialized && config.featureStore.initialized();
+  }
+
+  @Override
+  public void close() throws IOException {
```

Comment:
I think this will work fine b/c the tasks are periodic- not continuously running. Worst case it will wait until the current task is done. I can look at killing it more eagerly if you prefer.